client/reader/controller.js

```diff
@@ -48,12 +43,6 @@ function renderFeedError( context, next ) {
 }
 
 const exported = {
```

Comment:
This `exported` object that's destructured on export -- would you mind unrolling it into a standard `export function prettyRedirects( context, next ) {` series? It looks like an artifact of a CommonJS migration.

Reply:
Good point, sure thing 😉

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/53274/commits/1815581d87f4dfb397a5a5b5760c78ae97dbedde.